recipe/yum_requirements.txt

```diff
@@ -0,0 +1 @@
+xorg-x11-server-Xorg
```

Comment:
@beckermr, do you know why X11 server gets added when we switch to `matplotlib-base`? Would assume it is not needed as Qt is not being used then.

Reply:
Some packages needed this (at test time I think), so I added it by default. If the PR passes without it, then you are good to go.

Reply:
Thanks @beckermr. I removed it and the checks pass.

Reply:
Ah ok. Cool that makes sense. Thanks for clarifying :)